libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.ts

```diff
@@ -887,7 +887,7 @@ export class SkyAutocompleteComponent implements OnDestroy, AfterViewInit {
 
   #updateAriaOwns(): void {
     if (this.inputDirective) {
-      this.inputDirective.setAriaOwns(this.#overlay?.id || null);
+      this.inputDirective.setAriaControls(this.#overlay?.id || null);
```

Comment:
The name of this method doesn't describe what it's doing. Is there a reason we switched to `aria-controls`?

Reply:
Updated the method name. See above for the `aria-controls` explanation.

Reply:
Good catch. Updated.